core/include/irods/private/http_api/openid.hpp

```diff
@@ -14,12 +14,34 @@
 
 namespace irods::http::openid
 {
+	enum class TokenType
+	{
+		Access,
+		ID
+	};
+
 	auto create_oidc_request(boost::urls::url_view _url)
 		-> boost::beast::http::request<boost::beast::http::string_body>;
 
+	/// Validates an OAuth 2.0 Access Token using the Introspection Endpoint.
+	/// See RFC 7662 on OAuth 2.0 Token Introspection for more details.
+	///
+	/// \returns An optional containing a nlohmann::json object if verification was successful. Otherwise,
+	///          an empty optional is returned.
```

Comment:
Can we be a bit more specific about what is in the `nlohmann::json` object? It appears to be the result of requesting an access token using the provided `_bearer_token`. Can we expect this output to be similar to that returned by `validate_using_local_validation`?

Reply:
Changed wording on both returns.